src/nvim/event/multiqueue.c

```diff
@@ -82,7 +82,7 @@ typedef struct {
   Event event;
   bool fired;
   int refcount;
-} SplitEvent;
+} MulticastEvent;  ///< Event present on multiple queues.
```

Comment:
We could instead add these fields to `Event` itself, and also do the handling in `multiqueue_process_events()`. But maybe it's better to be explicit.

Reply:
Perhaps if this becomes more common, now this is just used in one specific event.